tests/console/rsync.pm

```diff
@@ -19,11 +19,33 @@ use warnings;
 use testapi;
 use utils;
 use version_utils qw(is_opensuse is_sle is_jeos);
+use Utils::Backends qw(is_svirt_except_s390x);
 
 sub run {
     my $self = shift;
     $self->select_serial_terminal;
     # try to install rsync if the test does not run on JeOS
+    # for the case that openssh is not installed
+    zypper_call("install -y openssh");
+    # create ssh key
+    assert_script_run("mkdir /root/.ssh/");
+    assert_script_run("ssh-keygen -t rsa -P '' -C 'root\@localhost' -f ~/.ssh/id_rsa");
+    assert_script_run("systemctl enable sshd");
+    assert_script_run("systemctl start sshd");
+    assert_script_run("systemctl status sshd");
+
+    # workaround for network issue on svirt backend, this can be removed when PR#15685 got merged
```

Comment:
These lines can be deleted safely now :) PR is merged

Reply:
As I said already, this module is not required for all other tests, so if we rely on it, it might be a risk if sshd is not started, that is the case on svirt backend.

Reply:
will add consoletest_setup anyway for a check, thanks!

Reply:
will remove them after I checked with further VRs, but actually this is already okay with current VRs.

Reply:
@Zaoliang can you mark bsc#1203626 as resolved?

Reply:
sure, I will.